modules/jooby-jakarta-validation/src/test/java/io/jooby/validation/app/PasswordsShouldMatchValidator.java

```diff
@@ -0,0 +1,15 @@
+package io.jooby.validation.app;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+
+public class PasswordsShouldMatchValidator implements ConstraintValidator<PasswordsShouldMatch, NewAccountRequest> {
+
+    @Override
+    public boolean isValid(NewAccountRequest request, ConstraintValidatorContext constraintContext) {
```

Comment:
Is there a way to access to jooby registry from `ConstraintValidatorContext`?????

Reply:
> Additionally, if the project utilizes some DI it is not a problem to inject any service/repo/etc. into custom validator

An example of that on docs will be nice.